src/utils/options.py

```diff
@@ -0,0 +1,313 @@
+    radctrl_to_exphan_identity: str = "RADCTRL_EXPHAN_IDEN"
+    radctrl_to_dsp_identity: str = "RADCTRL_DSP_IDEN"
+    radctrl_to_driver_identity: str = "RADCTRL_DRIVER_IDEN"
+    radctrl_to_brian_identity: str = "RADCTRL_BRIAN_IDEN"
+    radctrl_to_dw_identity: str = "RADCTRL_DW_IDEN"
+    driver_to_radctrl_identity: str = "DRIVER_RADCTRL_IDEN"
+    driver_to_dsp_identity: str = "DRIVER_DSP_IDEN"
+    driver_to_brian_identity: str = "DRIVER_BRIAN_IDEN"
+    driver_to_mainaffinity_identity: str = "DRIVER_MAINAFFINITY_IDEN"
+    driver_to_txaffinity_identity: str = "DRIVER_TXAFFINITY_IDEN"
+    driver_to_rxaffinity_identity: str = "DRIVER_RXAFFINITY_IDEN"
+    mainaffinity_to_driver_identity: str = "MAINAFFINITY_DRIVER_IDEN"
+    txaffinity_to_driver_identity: str = "TXAFFINITY_DRIVER_IDEN"
+    rxaffinity_to_driver_identity: str = "RXAFFINITY_DRIVER_IDEN"
+    exphan_to_radctrl_identity: str = "EXPHAN_RADCTRL_IDEN"
+    exphan_to_dsp_identity: str = "EXPHAN_DSP_IDEN"
+    dsp_to_radctrl_identity: str = "DSP_RADCTRL_IDEN"
+    dsp_to_driver_identity: str = "DSP_DRIVER_IDEN"
+    dsp_to_exphan_identity: str = "DSP_EXPHAN_IDEN"
+    dsp_to_dw_identity: str = "DSP_DW_IDEN"
+    dspbegin_to_brian_identity: str = "DSPBEGIN_BRIAN_IDEN"
+    dspend_to_brian_identity: str = "DSPEND_BRIAN_IDEN"
+    dw_to_dsp_identity: str = "DW_DSP_IDEN"
+    dw_to_radctrl_identity: str = "DW_RADCTRL_IDEN"
+    dw_to_rt_identity: str = "RT_DW_IDEN"
+    rt_to_dw_identity: str = "DW_RT_IDEN"
+    brian_to_radctrl_identity: str = "BRIAN_RADCTRL_IDEN"
+    brian_to_driver_identity: str = "BRIAN_DRIVER_IDEN"
+    brian_to_dspbegin_identity: str = "BRIAN_DSPBEGIN_IDEN"
+    brian_to_dspend_identity: str = "BRIAN_DSPEND_IDEN"
+
+    def parse_config(self):
+        # Read in config.ini file for current site
+        if not os.environ["BOREALISPATH"]:
+            raise ValueError("BOREALISPATH env variable not set")
+        if not os.environ['RADAR_ID']:
+            raise ValueError('RADAR_ID env variable not set')
+        path = f'{os.environ["BOREALISPATH"]}/config/' \
+               f'{os.environ["RADAR_ID"]}/' \
+               f'{os.environ["RADAR_ID"]}_config.ini'
+        try:
+            with open(path, 'r') as data:
+                raw_config = json.load(data)
+        except IOError:
+            print(f'IOError on config file at {path}')
+            raise
+
+        # Initialize all options from config file
+        self.site_id = raw_config['site_id']
+        self.main_antenna_count = int(raw_config["main_antenna_count"])
+        self.intf_antenna_count = int(raw_config["intf_antenna_count"])
+
+        # Parse N200 array and calculate which main and intf antennas operating
+        self.num_n200s = 0
+        self.n200_addrs = []
+        self.main_antennas = []
+        self.intf_antennas = []
+        for n200 in raw_config["n200s"]:
+            rx = bool(n200["rx"])
+            tx = bool(n200["tx"])
+            rx_int = bool(n200["rx_int"])
```

Comment:
I don't know that this will work. These fields are strings, yes? I just tested in a console and got this:

```python
>>>bool('true')
True
>>>bool('false')
True
>>>bool('False')
True
```

Reply:
I believe the fields are actually bools within JSON, and are read into python as bools. So the extra `bool()` casting is redundant.

Just tested within the class and confirmed they are read in as bools.